ci_framework/roles/hive/README.md

```diff
@@ -0,0 +1,27 @@
+# Role: hive
+This role can be used to claim clusters from a clusterpool.
+It assumes that oc > 4 is installed and configured.
+
+## Privilege escalation
+None
+
+## Parameters
+* `cifmw_use_hive`: Defaults to false.
+* `cifmw_hive_platform`: (Required), the platform to install OpenShift. It
+  could be a cloud provider or baremetal. Supported values are ('openstack',
+  'baremetal').
```

Comment:
where is that enforced i expected to find a check somewhere if we are saying these are the only supported values

Reply:
ack, added an asserts task for all the vars that are stated as required in the readme